### DISCUSSION TITLE
Redlining Text Drehung

### DISCUSSION THREAD

**Discussion by stpgis** [e-netze/webgis-community#174] [Ideas]
Beim Redlining Text wäre es für uns nützlich auch eine Drehung des Textes (könnte durch Eingabe einer Gradangabe / Winkels erfolgen) zu haben.
Bspw. für Beschriftungen von neuen Straßen.

**Comment by gis-eni** [comment]
Ist leider nicht ganz so easy zu implementieren. Bleibt vorerst einmal ein Verbesserungsvorschlag.
Für nicht stellt sich noch die Frage, ob Redlining das richtige Werkzeug ist. Warum die Straßennamen nicht gleich in eine Datenbank editieren? Redlining sollte ja eher etwas "temporäres" sein, zum Ausdrucken etc...

**Reply by stpgis** [reply]
Es ist nur temporär für den Sachbearbeiter zur Erstellung eines Antrages beim zuständigen Gremium. Die eigentlichen Eintragungen in Datenbanken GIP und DKM machen dann andere Mitarbeiter erst später. Die Sachbearbeiter hatten diese Möglichkeit beim alten Programm.
![grafik](https://github.com/gis-eni/webgis-forum/assets/171917921/f744ab8f-f0e0-40a9-b345-3a4d6f6bbe6a)
